presto-main/src/test/java/com/facebook/presto/type/TestJsonOperators.java

```diff
@@ -56,7 +56,16 @@ public void testCastToBigint()
     }
 
     @Test
-    public void testCastFromIntegrals()
+    public void testTypeConstructor()
+            throws Exception
+    {
+        assertFunction("JSON '123'", JSON, "123");
+        assertFunction("JSON '[4,5,6]'", JSON, "[4,5,6]");
+        assertFunction("JSON '{ \"a\": 789 }'", JSON, "{\"a\":789}");
+    }
+
+    @Test
+    public void testCastFromBigint()
```

Comment:
Mistake while resolving conflicts. Fixed in 4c22c0ce667852dd2eac5447f8dbddea12fbe54d.